test/util/fake-xml-http-request-test.js

```diff
@@ -1086,14 +1107,15 @@ if (typeof window !== "undefined") {
                 assert.isFalse(this.xhr.sendFlag);
             });
 
-            it("dispatches readystatechange event if sent before", function () {
+            it("does not dispatch readystatechange event if sent before", function () {
                 this.xhr.open("GET", "/");
```

Comment:
By "sent" you mean the actual event, not the `xhr.send()` call being triggered twice, right?